package.json

```diff
@@ -42,42 +41,24 @@
     "@types/react": "^18.0.0"
   },
   "devDependencies": {
-    "@babel/preset-react": "^7.16.7",
     "@changesets/changelog-github": "^0.4.6",
     "@changesets/cli": "^2.24.2",
-    "@types/fs-extra": "^9.0.13",
-    "@types/wait-on": "5.3.1",
-    "algoliasearch": "4.12.1",
-    "axios": "^0.24.0",
-    "chalk": "5.0.1",
+    "@local/eslint-config": "*",
```

Comment:
All deps were moved to `@local/package-chores`. `@local/eslint-config"` was added here because we still have a top-level `eslint` call. To be changed in https://github.com/blockprotocol/blockprotocol/pull/551